SixTrack/sixtrack.s

```diff
@@ -18738,7 +18738,9 @@ cc2008
      &'ERROR: length el(j) (elens is treated as thin element), '//
      &' and first and second field have to be zero: el(j)=ed(j)=ek(j)'//
      &'=0, while el(',j,')=',el(j),', ed(',j,')=',ed(j),', ek(',j,
-     &')=',ek(j),'.'
+     &')=',ek(j),'. Please check you input in the single element '//
+     &'definition of your HEL. All values except for the type need to'//
+     &'be zero.'
                call prror(-1)
```

Comment:
Now the error message becomes:
` ERROR: length el(j) (elens is treated as thin element),  and first and second field have to be zero: el(j)=ed(j)=ek(j)=0, while el(         108 )=   0.0000000000000000      , ed(         108 )=   0.0000000000000000      , ek(         108 )=   1.0000000000000000      . Please check you input in the single element definition of your HEL. All values except for the type need tobe zero.`

Good except for `tobe` :) Could you add the space?